robottelo/hosts.py

```diff
@@ -569,55 +568,11 @@ def install_katello_host_tools(self):
         if result.status != 0:
             raise ContentHostError('Failed to install katello-host-tools')
 
-    def install_katello_ca(self, satellite):
-        """Downloads and installs katello-ca rpm on the content host.
-
-        :param str satellite: robottelo.hosts.Satellite instance
-
-        :return: None.
-        :raises robottelo.hosts.ContentHostError: If katello-ca wasn't
-            installed.
-        """
-        warnings.warn(
-            message='The install_katello_ca method is deprecated, use the register method instead.',
-            category=DeprecationWarning,
-            stacklevel=2,
-        )
-        self._satellite = satellite
```

Comment:
Since you are removing the logic that ensures the connection between the `Satellite` host and the `ContentHost` object, I would like to see it being put in `register` and `api_register`

I am aware of the fact that ContentHost can be registered to Satellite **or** Capsule, but that is not something I would worry about too much at the moment.

Reply:
Good catch, I'll take a look where to add it.